tools/src/test/java/org/apache/kafka/tools/AclCommandTest.java

```diff
@@ -303,7 +294,7 @@ public void testUseBootstrapServerOptWithBootstrapControllerOpt() {
     @Test
     public void testUseWithoutBootstrapServerOptAndBootstrapControllerOpt() {
         assertInitializeInvalidOptionsExitCodeAndMsg(
-                Collections.emptyList(),
+                Collections.singletonList(ADD),
```

Comment:
Actually if you don't provide any arguments on the CLI, just the help message is printed.

The previous test was expecting a different behavior as `maybePrintHelpOrVersion()` used to be called in `main()` while I moved it to `AclCommandOptions.checkArgs()` for consistency with the other tools, and consistency with the CLI behavior.